docs/glossary.rst

```diff
@@ -15,6 +15,16 @@ Glossary
       syntax used to define the structuring of schemas. Common DDL
       statements include ``CREATE``, ``DROP``, and ``ALTER``.
 
+   link
+      Link items define a specific relationship between two object types. Link
+      instances relate one object to one or more different objects.
+
+      Comparing to SQL link is a way to define relationship, foreign key,
```

Comment:
Again, drawing parallels to SQL here isn't necessary, we can concentrate this on a separate page.